manage-mysql-user.py

```diff
@@ -485,14 +504,21 @@ def handler(event, context):
                 mysql_master_password_source,
                 mysql_master_password_source_type,
             )
-            execute_statement(
-                "GRANT {} ON `{}`.* to '{}'@'%';".format(
-                    privileges, database, mysql_user_username
-                ),
-                mysql_master_username,
-                mysql_master_password_source,
-                mysql_master_password_source_type,
-            )
+            for privilege_table in event["privileges"].split(", "):
+                privilege = privilege_table.split(":")[0]
+                table = (
+                    privilege_table.split(":")[1]
+                    if len(privilege_table.split(":")) is 2
+                    else "*"
```

Comment:
Does this mean if you do not specify a table, it's by default, all tables?

Reply:
Good question, yep, added a clarification in Readme.